packages/run-protocol/src/reserve/assetReserve.js

```diff
@@ -208,14 +213,15 @@ const start = async (zcf, privateArgs) => {
     zcf.makeInvitation(addCollateralHook, 'Add Collateral');
 
   const { brand: runBrand } = await E(runMint).getIssuerRecord();
-  /** @type {SubscriptionRecord<MetricsNotification>} */
-  const { publication: metricsPublication, subscription: metricsSubscription } =
-    makeSubscriptionKit();
 
   // shortfall in Vaults due to liquidations less than debt. This value can be
   // reduced by various actions which burn RUN.
   let shortfallBalance = AmountMath.makeEmpty(runBrand);
 
+  /** @type {import('../contractSupport.js').MetricsSubscriptionKit<MetricsNotification>} */
+  const { metricsPublication, metricsSubscription } =
+    makeMetricsSubscriptionKit(privateArgs.storageNode, privateArgs.marshaller);
+
```

Comment:
```suggestion
  /** @type {import('../contractSupport.js').MetricsSubscriptionKit<MetricsNotification>} */
  const { publisher: metricsPublisher, subscriber: metricsSubscriber } =
    makeStoredPublisherKit(privateArgs.storageNode, privateArgs.marshaller, 'metrics');

```

Reply:
I think this is the third assertion of your point in https://github.com/Agoric/agoric-sdk/pull/5613/files/33c4924387a82e068418621a29fad091bdb679e5#r903497158 . I'm confused why you're writing out the code here. It was clear enough what you're asking for. I gave my counterpoints there.

Reply:
> I'm confused why you're writing out the code here.

To make it easy to hit `Add suggestion to batch` if you elect to make the change.